filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -1,142 +1,111 @@
+:has_module_steps:
+:modulename: system nginx mysql
```

Comment:
same here, not sure I fully understand why these 3?

Reply:
@ruflin See my comment above.